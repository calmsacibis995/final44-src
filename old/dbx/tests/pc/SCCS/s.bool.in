h15097
s 00009/00000/00000
d D 5.1 90/06/25 15:36:40 bostic 1 0
c date and time created 90/06/25 15:36:40 by bostic
e
u
U
t
T
I 1
stop at 4
run
print b
step
print b
assign b = false
print b
assign b = true
print b
E 1
