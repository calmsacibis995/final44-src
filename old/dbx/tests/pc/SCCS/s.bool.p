h15305
s 00006/00000/00000
d D 5.1 90/06/25 15:36:43 bostic 1 0
c date and time created 90/06/25 15:36:43 by bostic
e
u
U
t
T
I 1
program booltest (input, output);
var b : boolean;
begin
    b := true;
    b := false;
end.
E 1
