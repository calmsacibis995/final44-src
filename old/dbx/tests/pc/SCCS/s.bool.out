h22071
s 00011/00000/00000
d D 5.1 90/06/25 15:36:41 bostic 1 0
c date and time created 90/06/25 15:36:41 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
[1] stop at 4
[1] stopped in program at line 4
    4       b := true;
false 
stopped in program at line 5
    5       b := false;
true 
false 
true 

E 1
